Writing-state-migrations.md

```diff
@@ -2,73 +2,26 @@ If your PR changes the properties of an exploration or state (or other structure
+5. Write a conversion function `_convert_states_v{old_state_version}_dict_to_v{old_state_version + 1}_dict` in _draft_upgrade_services.py_ that makes appropriate upgrades to data that resides in `ExplorationChange` lists. Here you have to check if the changes that you do in your conversion function affect the `draft_change_list`, if yes, then make sure to update the drafts or raise `InvalidDraftConversionException` which will drop the changes in the drafts (this needs to be done in cases where changes to drafts would be too complicated). If no changes need to be done to drafts you can simply return the `draft_change_list`.
+6. You would require to make some changes to the existing test files according to the changes you are doing in your conversion function. You have to edit the test functions present for the previous conversion functions in the `exp_domain_test.py` file to update the latest schema version. Also you have to update the schema of core/tests/data/oppia-ThetitleforZIPdownloadhandlertest!-v2-gold.zip file.
 
-8. Create a PR. If the tests fail, try resolving the test issues.
-9. Once your PR is finalized, file a one off job request for the ExplorationMigrationAuditJob using this [form](https://docs.google.com/forms/d/e/1FAIpQLSfvYWscAn18ok06An1oQ54h1VmBHfCX8uuuV01kIvY9WX0-Ug/viewform). The job tests a migration by running your conversion function on the dicts of existing exploration models and validating that the migration will be successful. It does this without committing the changes to the datastore.
-10. Fix any issues or errors from the audit job above.
-11. Get your migration PR merged.
-12. Once your PR is merged, please submit a request using this [form](https://docs.google.com/forms/d/e/1FAIpQLSfvYWscAn18ok06An1oQ54h1VmBHfCX8uuuV01kIvY9WX0-Ug/viewform) to run this migration in production. Before submitting this request, please ensure that the migration has already been tested manually on your local machine, passed code review, and been merged into develop.
+7. Create a PR. If the tests fail, try resolving the test issues.
+8. Once your PR is finalized, file a one request for the `AuditExplorationMigrationJob` and `MigrateExplorationJob` using this [form](https://docs.google.com/forms/d/e/1FAIpQLSfvYWscAn18ok06An1oQ54h1VmBHfCX8uuuV01kIvY9WX0-Ug/viewform). The job tests a migration by running your conversion function on the dicts of existing exploration models and validating that the migration will be successful. Make sure to mention to only run `MigrateExplorationJob` when the `AuditExplorationMigrationJob` is successful. The audit job does not commit changes to the datastore. After successful testing you can get you PR merged.
```

Comment:
```suggestion
8. Once your PR is finalized, file a one request for the `AuditExplorationMigrationJob` and `MigrateExplorationJob` using this [form](https://docs.google.com/forms/d/e/1FAIpQLSfvYWscAn18ok06An1oQ54h1VmBHfCX8uuuV01kIvY9WX0-Ug/viewform). The job tests a migration by running your conversion function on the dicts of existing exploration models and validating that the migration will be successful. Make sure to mention to only run `MigrateExplorationJob` when the `AuditExplorationMigrationJob` is successful. After successful testing you can get you PR merged.
```

Maybe instead of linking the form directly link the wiki page for testing the jobs.

Reply:
Do you refer to the Apache beam wiki here?

Reply:
This one https://github.com/oppia/oppia/wiki/Testing-jobs-and-other-features-on-production

Reply:
Updated